warehouse/query-core/src/test/java/datawave/query/iterator/FieldIndexIntegrationTest.java

```diff
@@ -79,9 +79,9 @@ public class FieldIndexIntegrationTest {
             put("FIELD_A", 2);
             put("FIELD_B", 2);
             put("FIELD_C", 2);
-            put("TLD_FIELD_A", 6);
-            put("TLD_FIELD_B", 26);
-            put("TLD_FIELD_C", 8);
+            put("TLD_FIELD_A", 4); // 3 record ids, 1 value
+            put("TLD_FIELD_B", 14); // 13 record ids, 1 value
+            put("TLD_FIELD_C", 5); // 4 record ids, 1 value
```

Comment:
What caused this assertion to change? Was the TLD aggregator not working before?

Reply:
no longer doing this.... I was attempting to fix the whole document/attribute equality mechanisms and it resulted in this change.  Undid all of that.